client/src/app/(overview)/url-store.ts

```diff
@@ -36,6 +37,18 @@ export const filtersSchema = z.object({
   [FILTER_KEYS[9]]: z.array(z.number()).length(2),
 });
 
+export const scorecardFiltersSchema = z.object({
+  availabilityOfExperiencedLabor: z.nativeEnum(PROJECT_SCORE).optional(),
+  availabilityOfAlternatingFunding: z.nativeEnum(PROJECT_SCORE).optional(),
+  coastalProtectionBenefits: z.nativeEnum(PROJECT_SCORE).optional(),
+  biodiversityBenefit: z.nativeEnum(PROJECT_SCORE).optional(),
+  financialFeasibility: z.nativeEnum(PROJECT_SCORE).optional(),
+  legalFeasibility: z.nativeEnum(PROJECT_SCORE).optional(),
+  implementationFeasibility: z.nativeEnum(PROJECT_SCORE).optional(),
+  socialFeasibility: z.nativeEnum(PROJECT_SCORE).optional(),
+  securityRating: z.nativeEnum(PROJECT_SCORE).optional(),
+});
+
```

Comment:
Unlike the the filters, that apply globally in the page and are shareable in the URL, `scorecardFiltersSchema` only applies to the scorescard table and it's not reflected in the URL, so let's move this `table/scorecard-priorization` better.